emscripten.py

```diff
@@ -142,7 +142,8 @@ def update_settings_glue(metadata, DEBUG):
 
 def apply_static_code_hooks(forwarded_json, code):
   code = shared.do_replace(code, '<<< ATINITS >>>', str(forwarded_json['ATINITS']))
-  code = shared.do_replace(code, '<<< ATMAINS >>>', str(forwarded_json['ATMAINS']))
+  if '_main' in shared.Settings.EXPORTED_FUNCTIONS:
```

Comment:
this doesn't match `HAS_MAIN`, which also handles dynamic linking and standalone mode.

perhaps we can set `HAS_MAIN` in python, before `src/jsifier.js` does so, and use it here?